src/markdown-pages/docs/managers/core-manager.md

```diff
@@ -3,12 +3,144 @@ path: "/docs/managers/core-manager"
 
 
+## <a name="dependencies"></a>Including the Manager in a test
+
+To use the Core Manager in a test you must import the _@CoreManager_ annotation into the test, as shown in the following example: 
+
+```
+@CoreManager
+public ICoreManager coreManager;
+```
+
+You also need to add the Manager dependency into the pom.xml file if you are using Maven, or into the build.gradle file if you are using Gradle. 
+
+If you are using Maven, add the following dependencies into the _pom.xml_ in the _dependencies_ section:
+
+```
+<dependency>
+<groupId>dev.galasa</groupId>
+<artifactId>dev.galasa.core.manager</artifactId>
+</dependency>
+```
+
+If you are using Gradle, add the following dependencies into ```build.gradle``` in the _dependencies_ closure:
+
+```
+dependencies {
+compileOnly 'dev.galasa:dev.galasa.core.manager'
+}
+```
+
+
+Finally, use the annotation that is provided by the Manager in the test. The annotation calls the Manager at runtime. In this example, the provided annotation is _@CoreManager_. 
+
+
+# <a name="configuring"></a>Configuring 
+
+The following are properties used to configure the Core Manager:
+
+## <a name="cps"></a>Configuration Properties
+
+<details>
+<summary>Resource String Pattern CPS Property</summary>
+
+| Property: | Resource String Pattern CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | core.resource.string.[length].pattern |
+| Description: | Sets the patterns of resources strings for a length of string.  The patterns are from the Galasa ResourcePoolingService which uses a homegrown syntax.  |
+| Required:  | No |
```

Comment:
Can you double check that description please, it doesn't read well for me

Reply:
have reworded